core/vm/actor/builtin/states/init_actor_state_raw.hpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "adt/address_key.hpp"
+#include "storage/hamt/hamt.hpp"
+#include "vm/actor/codes.hpp"
+
+namespace fc::vm::actor {
+  struct InitActorStateRaw {
```

Comment:
What is a destination of InitActorStateRaw?
It is inconsistent with other states